src/macros.rs

```diff
@@ -91,6 +91,22 @@ macro_rules! impl_pretty_debug {
     };
 }
 
+macro_rules! impl_non_secure_erase {
+    ($thing:ident, $target:tt, $value:expr) => {
+        impl $thing {
+            /// This function attempts to erase the contents of the underlying array.
+            /// Note, however, that since this type is `Copy`, ensuring that the contents
+            /// of this array don't end up elsewhere in memory is very subtle. For more
+            /// discussion on this, please see the documentation of the
+            /// [`zeroize`](https://docs.rs/zeroize) crate.
```

Comment:
```suggestion
           /// Attempts to erase the contents of the underlying array.
            ///
            /// Note, however, that since this type is `Copy`, ensuring that the contents
            /// of this array don't end up elsewhere in memory is very subtle. For more
            /// discussion on this, please see the documentation of the
            /// [`zeroize`](https://docs.rs/zeroize) crate.
```